### PR TITLE
Update compliance_monitoring.md

### PR DESCRIPTION
**Pull request by veraort**
I made a new section on the compliance monitoring and measurement page with the information I had collected from our seminar and what Markus Englund had already compiled in the issue #364 and #382. I have therefore simply listed Markus Englund and myself as contributors. I hope you agree with this @jmenglund.

### CODE REVIEW

**Comment by floradanna** [comment]
@veraort Thank you very much for this important content.
I am going to re-format a bit the layout and the style of the text, according to RDMkit way of writing.

I would like to add a practical example about the problem "How can I ethically access genetic resources of another country?". So, could you tell me if the following example would be appropriate or propose some other examples please?

Example: A researcher wants to re-use genetic data from an organism that is present only in a foreign country (animal or plant, etc) for academic research purpose (the research might generate an opportunity for intellectual property).

**Comment by veraort** [comment]
Dear @floradanna 
Thanks for the reply. It's nice that the content helps. Add a practical example is a great idea and it is already a suitable example. I would rephrase it a bit or add a second example, because it does not only concern the "re-use" case and Compliance measurements are added already under the first part "Planning - Problems to be addressed at this stage".

A researcher wants to re-use _or access new_ genetic data from an organism that is present only in a foreign country (animal or plant, etc) for academic research purpose (the research might generate an opportunity for intellectual property).

or

"A researcher from a country, which is party to the Nagoya Protocol, wants to access or re-use genetic resources and the associated data for research and use after 12 October 2014  from a foreign provider country. "

But maybe you can think of something better. But I would have both cases in there - re-use and new access.

Maybe we could also add this publication: "Explanation of the Nagoya Protocol on Access and Benefit Sharing and its implication for microbiology PMID: 28086069 DOI: 10.1099/mic.0.000425 " because they add "Generic Guidance: What biologists must do"

**Comment by floradanna** [comment]
Great, thank you for the suggestions!
